bundles/org.openhab.binding.hue/src/main/resources/ESH-INF/thing/Group.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="hue" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Generic thing for Groups/Rooms -->
+	<thing-type id="group">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge" />
+		</supported-bridge-type-refs>
+
+		<label>Hue Group</label>
+		<description>A group of lights or a room that could be switched on and off.</description>
+
+		<channels>
+			<channel id="switch" typeId="switch" />
+			<channel id="color_temperature" typeId="color_temperature" />
+			<channel id="brightness" typeId="brightness" />
+			<channel id="color" typeId="color" />
+		</channels>
+
+		<representation-property>groupId</representation-property>
+
+		<config-description>
+			<parameter name="groupId" type="text">
+				<label>Group ID</label>
+				<description>The group identifier identifies one certain hue group or room.</description>
+				<required>true</required>
+			</parameter>
+			<parameter name="fadetime" type="integer" min="0" step="100">
+				<label>Fade Time</label>
+				<description>Fade time in ms for changing values</description>
```

Comment:
```suggestion
				<description>Fade time for changing values</description>
```

Reply:
Changed.

Reply:
I revert this one becauyse Paper UI is not displaying th eunit, so without this information in the description, the user will not know what is the unit of this configuration setting.

Reply:
For your information: https://github.com/eclipse/smarthome/issues/3974

Reply:
I know that some UIs (e.g. Paper UI) - which will be removed in OH3 - do not show this "unit" but we prefer to not have it inside the description.

Reply:
Without the unit in the description, the user will not know how to adjust the setting.
Hopefully, the unit is inside the description in every existing bindings.
I would suggest to change thjat when OH3 is released, not before. And this will require to change more or less every bindings.